frame/alliance/src/lib.rs

```diff
@@ -603,7 +603,7 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			proposal_hash: T::Hash,
 			#[pallet::compact] index: ProposalIndex,
-			#[pallet::compact] proposal_weight_bound: Weight,
+			proposal_weight_bound: Weight,
```

Comment:
`#[pallet::compact]` is removed because `Weight` is no longer a simple typecast to `u64`?

Reply:
i dont think you can have compact on a struct? or can you?

Reply:
The struct cannot itself be compact, only the fields inside it. If i made the fields inside it compact, it would change how weight is encoded at the moment, which would be more breaking than we need.

It should be a follow up.

Reply:
Fixed actually with a suggestion by @bkchr here: https://github.com/paritytech/substrate/issues/12149